### DISCUSSION TITLE
List installed packages?

### DISCUSSION THREAD

**Discussion by andrewheiss** [posit-dev/positron#3762] [Q&A]
Is there a way to replicate RStudio's packages panel?

<img width="676" alt="image" src="https://github.com/posit-dev/positron/assets/73663/b665f1d4-b3a2-4082-b45d-50f8b509b0a6">

I've found this to be super useful when teaching R and I always direct students to this panel to install packages. And helpful in general for quickly seeing which versions of things I have installed, which ones have updates available, and so on.

In this issue (https://github.com/posit-dev/positron/issues/1369), there's a discussion about making it possible to see which packages are currently loaded in a session. I'm curious if there's currently a way to see which packages are *installed* but not necessarily loaded.

**Comment by juliasilge** [comment]
We do _not_ currently have a way to do this. We have some discussion about how we might implement such a feature in #1391 if you would like to add any additional context or detail.

**Comment by pander21** [comment]
Any updates on this in 2025?

**Reply by ntluong95** [reply]
You can find a user's extension here https://github.com/posit-dev/positron/discussions/7129

I have also bundled useful extensions for Positron, including extensions to list installed packages of R and Python here https://open-vsx.org/extension/ntluong95/positron-data-science-pack

**Comment by kv9898** [comment]
I made an [extension](https://open-vsx.org/extension/kv9898/positron-r-package-manager) exactly for this purpose:

<img width="414" height="548" alt="image" src="https://github.com/user-attachments/assets/8dd7e60b-dc00-46ce-bc40-32ba15368271" />

@ntluong95 also made one for Python: https://open-vsx.org/extension/ntluong95/positron-python-package-manager

**Comment by kategorically** [comment]
2 years later - any progress on this?

**Reply by juliasilge** [reply]
Yep! We have work in flight which you can see in https://github.com/posit-dev/positron/issues/11214 and the issues linked there. This will be available for users to try out in the 2026.04 release coming approximately next Monday. (It's already available in the dailies.)
